services/QuillLMS/app/views/blog_posts/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render partial: 'pages/shared/teacher_center_navbar', locals: {active_tab: @topic || BlogPost::ALL_RESOURCES } if (!current_user || current_user&.role == 'teacher') %>
+<%= render partial: 'pages/shared/teacher_center_navbar', locals: {active_tab: @topic || BlogPost::ALL_RESOURCES } if (!current_user || current_user&.role != 'student') %>
```

Comment:
I think this just got missed when we were doing the sweep for `.role == 'teacher'` - I might change it to just be `current_user&.teacher?` so we can stick with the pattern of using those methods instead of explicitly checking the value (the `teacher?` method will return `true` for both admins and teachers).

Reply:
sounds good! I'll update